pkgs/development/python-modules/cyclonedx-python-lib/default.nix

```diff
@@ -46,13 +47,21 @@ buildPythonPackage rec {
   checkInputs = [
     jsonschema
     lxml
-    pytestCheckHook
+    xmldiff
   ];
 
   pythonImportsCheck = [
     "cyclonedx"
   ];
 
+ checkPhase = ''
+   runHook preCheck
+   # Tests require network access
+   rm tests/test_output_json.py
+   ${python.interpreter} -m unittest discover -s tests -v
+   runHook postCheck
+ '';
```

Comment:
Why python unittest vs pytest? pytest should run unittest tests OK.

Reply:
With 2.2.0 I'm no longer able to get the test to pass with `pytestCheckHook`. Not sure if it's a path issue or something else. I will re-check with the next release.